src/DataTable/Pagination.tsx

```diff
@@ -0,0 +1,478 @@
+import {ChevronLeftIcon, ChevronRightIcon} from '@primer/octicons-react'
```

Comment:
I'm a little worried about keeping this[^1] in sync with our exported [Pagination](https://primer.style/react/Pagination) component.

Do you think it makes sense to do either of these?
- Introduce a new pagination component for use-cases like these
- Do a follow up PR where we update our existing Pagination component to have some of the features you've added here

[^1]: I'm only talking about the page numbers and prev/next buttons. I don't think we have other needs for the "1-10 of 1000" bit yet.

Reply:
I definitely think there is something there where we could merge the two components, in particular to share appearance and behavior. It'd be great if we could figure out how to allow customization of the inner pages to allow for the different navigation use-cases (button vs a). I have no idea how to expose this kind of stuff to an end user without coming up with some kind of naming scheme between the two though 🤔

Reply:
Should we create an issue to figure this out? Want to discuss it during the next Primer patterns working session?

Reply:
Definitely! Made one over at: https://github.com/github/primer/issues/2164 feel free to add to it if I missed anything 👀